scripts/lsf/bash/run.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 #
-# Copyright © 2018 IBM Corporation
+# Copyright © 2019 IBM Corporation
```

Comment:
This is a minor... I guess, if the file is not entirely new, the year has to show either `2018-2019` or `2018,2019`